config/formats.js

```diff
@@ -1213,7 +1213,7 @@ let Formats = [
 
 		mod: 'gen4',
 		ruleset: ['Standard', 'Baton Pass Clause'],
-		banlist: ['Uber', 'Sand Veil', 'Soul Dew'],
+		banlist: ['Uber', 'Sand Veil', 'Soul Dew', 'Swinub ++ Snow Cloak', 'Piloswine ++ Snow Cloak', 'Mamoswine ++ Snow Cloak'],
```

Comment:
Should these have been one `+` instead of two? As it stands, this bans any of those three users if there is another Snow Cloak user on the team.

Reply:
A lot of people seem to mix up `+` and `++`. We should probably replace `++` with something clearer?

Reply:
Especially since the current syntax seems to imply you can mix `+` and `++` when you really can't.

Reply:
I wonder if a keyword meaning "in the same team" would help

`Drizzle + Swift Swim in the same team`

But some way to make that more concise...

Reply:
But who knows, maybe it _should_ be that long? It has very few uses, mostly just a Gen 5 complex ban.

Reply:
Replace + with @. “Mamoswine @ Snow Cloak” for example makes perfect sense to me. And then probably change ++ to +.

Reply:
"Shell Smash @ Baton Pass" doesn't really make sense to me, though.

Reply:
why not replace `++` with `*`? i dont have any reasoning that makes sense, but its another mathematical method

Reply:
`Drizzle + Swift Swim in the same team` is my preference, because the point is that it's very rarely used and would benefit from the additional clarity.

Reply:
I'm not too bothered by it I guess

Reply:
What about ``&`` instead of ``++``?

Reply:
I don't think that'll fix the "get people to realize they should be using `+` and not `&`" problem.